build-ps/percona-server.spec

```diff
@@ -495,10 +500,15 @@ datadir=$(/usr/bin/my_print_defaults server mysqld | grep '^--datadir=' | sed -n
 %else
 /sbin/chkconfig --add mysql
 %endif
-# We need this because we don't provide my.cnf on centos 6
-# and the default system one doesn't have info needed
-%if 0%{?rhel} < 7
-if [ $1 -eq 1 ]; then
+if [ ! -f /etc/my.cnf ]
+then
+  update-alternatives --install /etc/my.cnf my.cnf "/etc/percona-server.cnf" 200
+else
+  echo " -------------"
+  echo "   *  The suggested my.cnf options and settings are in percona-server.conf.d/my.cnf"
```

Comment:
should this be: "*  The suggested my.cnf options and settings are in /etc/percona-server.conf.d/my.cnf"?